c++/src/kj/common.h

```diff
@@ -1928,6 +1928,20 @@ class ArrayPtr: public DisallowConstCopyIfNotConst<T> {
     return true;
   }
 
+  //Lexographic difference function can be overriden for the type by defining <=> on your type
```

Comment:
nits: //<space>

also check around for comment style - it comes _after_ the line. (kj style guide)